jscomp/others/js_types.ml

```diff
@@ -75,9 +80,11 @@ let classify (x : 'a) : tagged_t =
   if ty = "function" then
     JSFunction (Obj.magic x) else
   if ty = "object" then
-    JSObject (Obj.magic x)
-  else
+    JSObject (Obj.magic x) else
+  if ty = "symbol" then
     JSSymbol (Obj.magic x)
+  else
+    JSBigInt (Obj.magic x)
```

Comment:
Anything better than "if everything else fails" one can use here?

Reply:
![Screenshot 2022-06-25 at 03 50 48](https://user-images.githubusercontent.com/7965335/175753802-1b112ce8-46e0-498d-9344-9fb88ff6cb5d.png)

How about following the same checks, with the last "if all else fails" then object.